dali/test/python/operator/test_arithmetic_ops.py

```diff
@@ -793,3 +799,42 @@ def test_prohibit_min_max():
                          " be used for truth evaluation in regular Python context."))
 def test_bool_raises():
     bool(DataNode("dummy"))
+
+
+def test_binary_ops_broadcasting():
```

Comment:
I think we should test a bit of failed bcasting, to see if the error messages make sense.

Reply:
:+1: